.github/workflows/cypress-test.yml

```diff
@@ -9,95 +9,45 @@ env:
   START_CMD: 'node ../scripts/opensearch_dashboards --dev --no-base-path --no-watch --opensearch_security.multitenancy.enable_aggregation_view=true'
   OPENSEARCH_SNAPSHOT_CMD: 'node ../scripts/opensearch snapshot'
   SPEC: 'cypress/integration/plugins/security-dashboards-plugin/aggregation_view.js,'
+  OPENSEARCH_VERSION: 3.0.0
+  PLUGIN_NAME: opensearch-security
+  PLUGIN_VERSION: 3.0.0.0
 
 jobs:
   tests:
-    name: Run aggregation view cypress test
+    name: Run Cypress tests
     runs-on: ubuntu-latest
     steps:
-      - name: Download OpenSearch Core
-        run: |
-          wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/3.0.0/latest/linux/x64/tar/builds/opensearch/dist/opensearch-min-3.0.0-linux-x64.tar.gz
-          tar -xzf opensearch-*.tar.gz
-          rm -f opensearch-*.tar.gz
-          
+      - name: Checkout Branch
+        uses: actions/checkout@v2
+      
       - name: Download OpenSearch Security Plugin
-        run: wget -O opensearch-security.zip https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/3.0.0/latest/linux/x64/tar/builds/opensearch/plugins/opensearch-security-3.0.0.0.zip
-        
-
-      - name: Run OpenSearch with plugin
+        run: wget --progress=bar:force:noscroll -O opensearch-security.zip https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/${{ env.OPENSEARCH_VERSION }}/latest/linux/x64/tar/builds/opensearch/plugins/${{ env.PLUGIN_NAME }}-${{ env.PLUGIN_VERSION }}.zip
```

Comment:
why `--progress=bar:force:noscroll` here?

Reply:
Hi @DarshitChanpura thanks for the review! We use this flag because the log will look like this https://github.com/RyanL1997/security-dashboards-plugin/actions/runs/3576589551/jobs/6014577673#step:7:837. This one is so hard to track.

Reply:
Lets replace this with `peternied/download-file@v2` in a fast-following PR, as these actions should support multi-platform which it does not